Solution1/Solution1.Tests/BL/Services/WeatherServiceTests.cs

```diff
@@ -119,5 +120,70 @@ public async Task GetForecastByCityNameAsync_ReturnedForecastWeatherDTO_Success(
 
             Assert.True(new CompareLogic().Compare(expectedWeatherDto, result).AreEqual);
         }
+
+        [Fact]
+        public async Task GetWeatherByArrayCityNameAsync_ReturnedDictionaryForWeatherResponseDTO_Success()
+        {
+            var cityName2 = "Paris";
+            var cityName3 = "AАА";
+            var cityName4 = string.Empty;
+            var weatherApiDto1 = new WeatherApiDTO() { CityName = cityName, TemperatureValues = new WeatherApiTempDTO() };
+            var weatherApiDto2 = new WeatherApiDTO() { CityName = cityName2, TemperatureValues = new WeatherApiTempDTO() };
+            
+            var listCityName = new List<string>() { cityName, cityName2, cityName3, cityName4 };
+            
+            var validationFailure = new List<ValidationFailure>() { new ValidationFailure("CityName", "Test validation error")};
+            var notValidResult = new ValidationResult(validationFailure);
+            var ValidResult = new ValidationResult(new List<ValidationFailure>());
+
+            SetWeatherApiServiceSettings(_weatherApiServiceMock, weatherApiDto1);
```

Comment:
_weatherApiServiceMock is a class field, why you pass it to the method

Reply:
+